client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -6,33 +6,48 @@
     :class="GreetDeviceStep[pageStep]"
   >
     <ms-wizard-stepper
-      v-show="pageStep > GreetDeviceStep.WaitForGuest && pageStep < GreetDeviceStep.Summary"
+      v-show="pageStep > GreetDeviceStep.WaitForGuest && pageStep < GreetDeviceStep.Summary && isLargeDisplay"
       :current-index="getStepperIndex()"
       :titles="['DevicesPage.greet.steps.hostCode', 'DevicesPage.greet.steps.guestCode']"
     />
     <ion-button
       slot="icon-only"
       @click="cancelModal()"
-      v-show="pageStep !== GreetDeviceStep.Summary"
       class="closeBtn"
+      v-if="pageStep !== GreetDeviceStep.Summary && isLargeDisplay"
     >
       <ion-icon
         :icon="close"
         class="closeBtn__icon"
       />
     </ion-button>
     <div class="modal">
-      <ion-header class="modal-header">
+      <ion-header
+        class="modal-header"
+        v-if="isLargeDisplay"
+      >
         <ion-title class="modal-header__title title-h2">
-          {{ $msTranslate(getTitleAndSubtitle().title) }}
+          {{ $msTranslate(getStep().title) }}
         </ion-title>
         <ion-text
           class="modal-header__text body"
-          v-show="getTitleAndSubtitle().subtitle"
+          v-show="getStep().subtitle"
         >
-          {{ $msTranslate(getTitleAndSubtitle().subtitle) }}
+          {{ $msTranslate(getStep().subtitle) }}
         </ion-text>
       </ion-header>
+      <small-display-step-modal-header
```

Comment:
`close button` should be hidden on the last step as it makes no sense (see what's done for large displays:

```
      class="closeBtn"
      v-if="pageStep !== GreetDeviceStep.Summary && isLargeDisplay"
```